### ISSUE TITLE
[Local-Preview] In-product nudge towards next installation steps (i.e. PoV reference Architecture)

### ISSUE THREAD

**Issue by lucasvaltl** [gitpod-io/gitpod#11381]
## Is your feature request related to a problem? Please describe
When a user successfully installed and seems to be using / happy with their installation, we should nudge them towards the logical next step in the user journey: to install Gitpod on K8s, likely via our PoV Reference architecture (https://github.com/gitpod-io/gitpod/issues/10622). 

## Describe the behaviour you'd like
- When a user has successfully installed and started 1 workspace, there should be a toast that asks them to go to the next installation method. 
- The amount of workspaces needed until we nudge should be configurable
- The nudge should point to the community license form, however with attribution (we should be able to track that a user came from a preview installation) - e.g. using an UTM link
- The same nudge should also be in the documentation.
- This only happens in the local preview

## Describe alternatives you've considered
- Only do this in the documentation - which might be overlooked.

## Additional context
- Internal [planning document](https://www.notion.so/gitpod/Local-Preview-Plans-f9c200ebfa154bf7a6ca3c550204aac4)
- Internal [release plan](https://www.notion.so/gitpod/Release-Plan-Local-Preview-a635787b23a84649967a64819e6be79f)

## Implementation Ideas:
- Can show a goodbye message ("thanks for trying Gitpod, go to this link for next steps") on terminate signal in the CLI (with condition "has opened a workspace")
- AND show this within the dashboard - see [internal discussion ](https://gitpod.slack.com/archives/C02EN94AEPL/p1657872926410249)
  - We can hide this behind a feature flag such that it only gets shown in local preview mode - see [this function](https://github.com/gitpod-io/gitpod/blob/fdc44f4beab2c0cdcac56cc71b19f2b64553418b/components/dashboard/src/utils.ts#L52-L59)
  - From @gtsiolis: I’d suggest to use a non-closable, info variant of the alert component variant placed at as a global page alert (see examples[[1](https://github.com/gitpod-io/gitpod/issues/9974#issuecomment-1125023447)][[2](https://www.figma.com/file/jorFpdvydV22oRc0YqKWGo/Add-phone-verification-for-new-users-%2311339?node-id=1402%3A711)]).
    - The content of the alert could inform users they are running a preview that is not intended production usage, and provide a call-to-action (CTA) to install with a community license, contact sales for buying an enterprise license, or later on start a free trial of the enterprise license.

**Comment by Pothulapati** [comment]
Merged https://github.com/gitpod-io/gitpod/pull/11434 which adds a warning alert in the dashboard! I've tried to do the terminate CLI option, but it's heavily dependent on which terminates first i.e script or the container itself, and couldn't get far.

Because the website nudge provides a direct path to action (as its through the browser), and always there, I'm thinking this is enough. WDYT?

(A release will be done this week, along with documentation being updated)
